src/Model/RatingRank.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace sodoff.Model;
+
+public class RatingRank {
+    [Key]
+    public int Id { get; set; }
+
+    public int CategoryID { get; set; }
+
+    public int? RatedEntityID { get; set; }
+    public string? RatedUserID { get; set; }
+
+    public int Rank { get; set; }
+
+    /// <summary>On a scale of 1-5</summary>
+    public float RatingAverage { get; set; }
+
+    public int TotalVotes { get; set; }
```

Comment:
we don't need `TotalVotes` if it's just the number of elements in `Ratings` collection